### ISSUE TITLE
Touch +Narrator: Narrator is announcing incorrect group name for popup window on navigate to close button

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3506]
**Describe the bug**

The New flyout is opening in its own window. The flyout traps focus, the rest of the window is an implied close button to enable light dismiss.
![image](https://user-images.githubusercontent.com/13142665/97392835-03a65b00-18a0-11eb-96da-3da39d48cd7d.png)

**Steps to reproduce the bug**

Pre-requisite:
- Turn on Narrator on a touch screen

1. Open Snip & Sketch
2. Navigate to “New” button and triple tap to expand it.
4. Swipe down and left to activate touch investigation. This moves Narrator focus to the close button of the popup window
5. Observe the Narrator announcement

Note:
- Same issue is observed for all the popup menus
- Issue is observed only using Touch + Narrator

**Expected behavior**

Narrator should announce the correct group name of popup window

**Screenshots**

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/25307087)
